src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,58 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_TYPE_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+    private static final String SUPPLY = "supply,";
+    private static final String BUY = "buy,";
+    private static final String RESULT = "result,";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String infoFromFile = readFromFile(fromFileName);
+        String report = createReport(infoFromFile);
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName))) {
+            bufferedWriter.write(report);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write to file", e);
+        }
```

Comment:
let's move this logic to a separate private method `writeToFile(String fileName, String content)`